Úloha J03

JakubMartak opened the pull request:
Tá tabuľka bola zlá, ako máme do pekla pracovať bez indexu!

JakubMartak committed:
Úloha J03

Tá tabuľka bola zlá, ako máme do pekla pracovať bez indexu!

ukf-michalkohutek commented:
Tabulka je ok, zly je len pristup. Index nepotrebujete na to, co po vas 
chcem. Na identifikaciu a mazanie riadku vam databazovy prikaz

SELECT * FROM person WHERE firstname = "krstnemeno";

Navyse, nikde nebolo povedane, ze tu tabulku nemozete upravovat. Kludne 
si do nej mozete pridat novy stlpec s ID,

MK

On 2020-04-03 20:18, JakubMartak wrote:
> Tá tabuľka bola zlá, ako máme do pekla pracovať bez indexu!
> -------------------------
> 
> YOU CAN VIEW, COMMENT ON, OR MERGE THIS PULL REQUEST ONLINE AT:
> 
>  https://github.com/ukf-michalkohutek/J03/pull/2 [1]
> 
> COMMIT SUMMARY
> 
>  	* Úloha J03
> 
> FILE CHANGES
> 
>  	* M
> AddressBookSQLite/src/AdressBookSQLite/FXMLTableViewController.java
> [2] (18)
>  	* M AddressBookSQLite/src/AdressBookSQLite/SQLiteJDBC.java [3] (31)
>  	* M AddressBookSQLite/src/AdressBookSQLite/fxml_tableview.fxml [4] 
> (2)
> 
> PATCH LINKS:
> 
>  	* https://github.com/ukf-michalkohutek/J03/pull/2.patch [5]
>  	* https://github.com/ukf-michalkohutek/J03/pull/2.diff [6]
> 
> --
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub [1], or unsubscribe 
> [7].
> 
> Links:
> ------
> [1] https://github.com/ukf-michalkohutek/J03/pull/2
> [2]
> https://github.com/ukf-michalkohutek/J03/pull/2/files#diff-797708a6da43cf42283747c127aadd53
> [3]
> https://github.com/ukf-michalkohutek/J03/pull/2/files#diff-dfed809e28dd2a3a125001a30722b106
> [4]
> https://github.com/ukf-michalkohutek/J03/pull/2/files#diff-f66f85205885cdc77c08182ecfd5b250
> [5] https://github.com/ukf-michalkohutek/J03/pull/2.patch
> [6] https://github.com/ukf-michalkohutek/J03/pull/2.diff
> [7]
> https://github.com/notifications/unsubscribe-auth/AOTHXCUZGVG5CGXLKGSZGX3RKYSAPANCNFSM4L4LGGRA

JakubMartak commented:
Nie som si istý, že či rozumiem. Mám ich identifikovať podľa krstného mena, ktoré je v textfielde, alebo ktoré je označené. A takých mien môže byť viac, nie. Tiež, prečo select *, prečo by ma malo zaujímať úplne všetko? Mám takto upraviť len update, alebo aj delete. Taktiež neviem, že či = "krstnemeno". To nie a nie zafungovať. Fungovalo až like "krstnemeno".

PS.: Do tabuľky som sa pôvodne pokúšal pridať ID, ale po dvoch hodinách som to vzdal.

JakubMartak committed:
J03 Mark2

Toto som dokázal vymyslieť, aby to fungovalo. Mohli by sme si niekedy dohodnúť chat?

JakubMartak committed:
Záverečná Úprava

ukf-michalkohutek commented:
ok
